config/jobs/kubernetes/sig-windows/windows-gce.yaml

```diff
@@ -82,7 +82,7 @@ periodics:
       - --test_args=--ginkgo.focus=\[Conformance\]|\[NodeConformance\]|\[sig-windows\] --ginkgo.skip=\[LinuxOnly\]|\[Serial\]|\[Feature:.+\] --minStartupPods=8
       - --test-cmd-args=--ginkgo.focus=\[Conformance\]|\[NodeConformance\]|\[sig-windows\] --ginkgo.skip=\[LinuxOnly\]|\[Serial\]|\[Feature:.+\] --minStartupPods=8
       - --test-cmd-args=--node-os-distro=windows
-      - --timeout=120m
+      - --timeout=180m
```

Comment:
I think you need to change the `interval` field up above too.

Reply:
oh, got it. But I am wondering whether it is actually allowed to have interval shorten than timeout. I assume multiple tests can be run concurrently?

Reply:
I'm guessing that with interval > timeout this will work fine, but I don't have any detailed knowledge of how these configs work. If something breaks, we'll fix it.